packages/builder/src/components/backend/DataTable/modals/EditRoles.svelte

```diff
@@ -101,14 +100,28 @@
     }
   }
 
+  const getRoleNameError = name => {
+    const hasUniqueRoleName = !otherRoles
```

Comment:
Should we worry about casing?

Reply:
I think we allow it to be case sensitive, we haven't compared the names before now (we already checked if the role name was "unique" but we allowed it to be case sensitive).